ui-modules/app-inspector/app/components/entity-tree/entity-node.less

```diff
@@ -75,10 +75,28 @@
         margin-left: 10px;
       }
     }
+    .entity-node-toggle-wrapper {
+      width: 38px;      
+      flex-grow: 0;
+    }
     .entity-node-toggle {
-      flex-shrink: 1;
+      width: 38px;      
       padding: 12px;
       cursor: row-resize;
     }
   }
+  .node-icon {
+    .pull-right();
```

Comment:
Why using `.pull-right()` mixin where the whole block is using flexbox? This is not right, should be one or the other. Based on the current setup, I think this should use flexbox only)

Reply:
as it's non-growable and the other field is growable, we don't need it at all, quite right